third_party_manifest.json

```diff
@@ -94,15 +94,15 @@
       "publisher": "New Relic Node.js agent team",
       "email": "nodejs@newrelic.com"
     },
-    "@newrelic/security-agent@0.0.4": {
+    "@newrelic/security-agent@1.0.0": {
       "name": "@newrelic/security-agent",
-      "version": "0.0.4",
-      "range": "0.0.4",
-      "licenses": "Apache-2.0",
+      "version": "1.0.0",
+      "range": "^1.0.0",
+      "licenses": "UNKNOWN",
```

Comment:
No license?

Reply:
It's the restricted pre-release New Relic license, a bit odd for a public NR repo. https://github.com/newrelic/csec-node-agent/blob/v1.0.0/LICENSE

Reply:
Generating this data from the automated tool which is setting license  to UNKNOWN. Although LICENSE.md file is provided along with security agent and We got this License from legal team.